### PR TITLE
Pull request for #154: Installation is unsuccessful every other time

### PR DESCRIPTION
**Pull request by DavidRitzwoller**
In conjunction with the changes I've committed [here](https://github.com/gslab-econ/template/pull/107), the pip 10 issues are fixed. The issue was that the requirements.txt installation uses egg, whereas pip uses wheel. Now gslab_python is installed with the wheel, removing the need to manually delete old versions of GSLab_tools implemented [here](https://github.com/gslab-econ/gslab_python/issues/122).

### CODE REVIEW

**Comment by DavidRitzwoller** [comment]
@stanfordquan I made the following changes to gslab_python
- I updated the README to instruct the reader how to install from the root of the directory i.e use `pip install .` rather than `python setup.py install clean`. I also added language clarifying why `python setup.py` install should not be called and when to use `python setup.py clean`.
- As mentioned [here](https://github.com/gslab-econ/gslab_python/pull/155#issue-199868025) I removed the code in setup.py which manually cleans `site-packages` of all gslab packages at the beginning of each run.

File: README.md [hunk 1]
```diff
@@ -42,10 +42,19 @@ say, a `sudo` command.
...
+```
+
+We do not reccommend that these packages be installed by executing
+```bash
+python setup.py install
+```
+This method of installation uses egg files rather than Wheels, which can cause conflicts with previous versions of `gslab_tools`. If this method of installation is executed, some files need to be removed from the directory with a `clean` argument. `clean` removes `/build`,`/dist`, and `GSLab_Tools.egg-info`, which are built upon installation. This argument can be called by executing 
+
+```bash
+python setup.py clean
```
Comment (stanfordquan) [inline README.md]: I think this has to be 

```
python setup.py install clean
```

right?

Comment (DavidRitzwoller) [reply README.md]: No, `python setup.py clean` just cleans the repository, including the install argument also reinstalls the packages. setup.py accepts the arguments install, test, and clean.

Comment (stanfordquan) [reply README.md]: Ah, got it. Thanks!

**Comment by stanfordquan** [comment]
Thanks @DavidRitzwoller. Can you please merge this into dev4.1.1? We can pull this in and I'll pull in the template branch once we pull in 4.1.1.

**Pull request merged**